.moban.yaml

```diff
@@ -1,6 +1,30 @@
 overrides: coala.yaml
 
 name: coala-quickstart
+contact: coala.analyzer@gmail.com
+description: A quickstart tool for coala
+current_version: 0.4.0
+version: 0.4.0
+build_version: 0.4.0
+package_module: coala_quickstart
+url: https://github.com/coala/coala-quickstart
+docs_dir: false
+
+maintainers: false
```

Comment:
why is this `False` 😅

Reply:
template logic requires it in order to use `maintainer_list`

Reply:
Ah I see.